app/views/groups/show.html.erb

```diff
@@ -0,0 +1,31 @@
+
+
+ <div class="category-show-container">
+
+    <%= link_to groups_path do %>
+      <i class="fa-solid fa-circle-chevron-left back-icon"></i>
+    <% end %>
+
+   <div class="category-show">
+      <p class="category-icon"> <%= @group.icon %> </p>
+      <p> <%= @group.name.capitalize %> </p>
+   </div>
+
+    <div>
+        <% @group.entities.each do |entity| %>
+
+        <div class="category-detail">
+             <div class="category-amount">
+                <p> <%= entity.name.capitalize %></p>
+                <p> $<%= entity.amount %></p>
+             </div>
+              <p> <%= entity.created_at%> </p>
```

Comment:
- [**Optional**] I think the date display on this page is not well formatted. To improve our user's experience, let's format the date. The date looks better in the screenshot below. I think we can also do the same.

![Screenshot from 2022-12-10 15-02-48](https://user-images.githubusercontent.com/63087491/206859216-4c4cfd7c-a9ff-485a-997f-4eb926910e02.png)